README.md

```diff
@@ -38,7 +38,7 @@ CGo-based library.
 
 ## Requirements
 
-- Go 1.18+
+- Go 1.20+
```

Comment:
This is a breaking change, maybe we need more discussion.

Reply:
this is from the required upgrade of github.com/klauspost/compress v1.17.9

Reply:
Usually, we want to be compatible with lower versions of Golang, but some libraries force us to upgrade to higher versions.
 
Using 1.20 is in accordance with Golang's release policy: https://go.dev/doc/devel/release

We will release 0.13.0, and I think we can make this change.

Reply:
Yes. According to the go release policy:

> Each major Go release is supported until there are two newer major releases.

The currently supported go version is 1.22 and 1.21. We are OK to upgrade the lowest supported go version here.